web/template/partial/stream/chat/chat.gohtml

```diff
@@ -39,6 +46,86 @@
                 </div>
             </div>
         </div>
+        <!-- Active Poll UI -->
+        <template x-if="activePoll && !admin">
+            <div class="z-50 absolute w-full">
+                <div class="text-white bg-sky-600 dark:bg-sky-700 text-center p-2">Active Poll</div>
+                <div class="shadow-lg p-4 bg-white dark:text-white dark:bg-secondary-lighter border-b dark:border-gray-800 rounded-b-lg">
+                    <p x-text="activePoll.question" class="text-sm font-semibold mb-3"></p>
+                    <template x-for="pollOption in activePoll.pollOptions" :key="pollOption.ID">
+                        <button class="flex items-center mb-1" @click="activePoll.selected = pollOption.ID" :disabled="activePoll.submitted != 0">
+                            <i class="" :class="(pollOption.ID == activePoll.selected || activePoll.submitted == pollOption.ID) ? 'fas fa-check-circle' : 'far fa-circle'"></i>
```

Comment:
Works great, good job 👍🏻 

A tiny thing I noticed: You used `==` and `!=` instead of `===` and `!==` a couple of times in chat.gothml. This might result in weird behaviour in some edge cases.

Reply:
Yes cleaner to compare types as well, I agree! :)